### ISSUE TITLE
The meaning of "flip" in smobj "e" lines

### ISSUE THREAD

**Issue by ixchow** [textiles-lab/smobj#1]
It appears that `knitout-to-smobj` and `smobj-to-yarns` may both have bugs around the meaning of "flipping" an edge connection. According to the readme:
```
#list of connections between faces:
# face#/edge#, both one-based; negative edges imply *reversing* the edge
# this allows non-orientable connections, needed for knitout
# (NOTE: on a nicely oriented mesh, all 'e' commands will have one negative edge)
e 1/1 2/-4
```

Which would seem to imply that in a simple situation like:
```
+---+
| A |
+---+
| B |
+---+
```
where `A` and `B` are both, say, `knit-to-right`, the edge between them *should* be marked flip. But it appears that, in fact, marking as flip results in an unexpected twist in the stitch:
```
L knit-to-right -l1 +y1 +l1 -y1
v -1 -0.61 0
v 1 -0.61 0
v 1 0 0
v -1 0 0
v -1 0 0
v 1 0 0
v 1 0.61 0
v -1 0.61 0
f 1 2 3 4
T 1
N 0
f 5 6 7 8
T 1
N 0
e 1/3 2/1 #<-- this should be "flipped" but it isn't
```

Result with the "unflipped" (wrong-according-to-README) e:
![image](https://user-images.githubusercontent.com/1418494/65398604-408f4c80-dd86-11e9-8a31-394db747efb6.png)

Result with the "flipped" (correct-according-to-README) e:
![image](https://user-images.githubusercontent.com/1418494/65398628-5bfa5780-dd86-11e9-9f10-516d50487b6c.png)

**Comment by ixchow** [comment]
It appears that `text-to-knitout` has also inherited this bug.

**Comment by ixchow** [comment]
Fixed in 606b15d and 0ab21a5; utilities and yarn generation are now consistent with the as-written definition.
